symplyphysics/laws/thermodynamics/efficiency_from_quantity_of_heat.py

```diff
@@ -0,0 +1,42 @@
+from sympy import Eq, solve
+from sympy.physics.units import joule
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    LessEq
```

Comment:
There is no such function in either sympy, or symplyphysics. Try your code before submission.

Reply:
so how to fix this?

Reply:
https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/thermodynamics/inner_energy_from_temperature.py - I tried to copy this because i havent enough experience with this project to solve my first task

Reply:
I mean the formula you've tried to implement. Where did you find it?

Reply:
In wiki, this formula refers to the expression of the efficiency of heat engines and is based on the second principle of thermodynamics. Just saw that there is no such formula and tried to implement it

Reply:
Give me a link.

Reply:
https://en.wikiversity.org/wiki/Thermodynamics/The_Second_Law_of_Thermodynamics

Reply:
I do not see an inequality like you are trying to implement.